src/shared/scrapers/PA/index.js

```diff
@@ -0,0 +1,299 @@
+    //   CODIGO: '010210', - probably a ZIP code?
+    //   CAMPO: '2017',
+    //   ID2: '1',
+    //   TIPO: ' ',
+    //   Cantidad: '', - "Quantity" (note these appear to be coronavirus-related fields, but they're not populated.
+    //   Hospitalizado: '', - "Hospitalized"
+    //   Aislamiento_domiciliario: '', - "Home quarantine"
+    //   Fallecido: '',  - "Deceased"
+    //   UCI: '', - "Intensive Care"
+    //   Recuperado: '', - "Recovered
+    //   ORIG_FID: '1'
+    // }
+
+    // Cache this.
+    /* const tests = */ await fetch.csv(this._testsUrl);
+    // Simple test counts (country level):
+    // [
+    //   {
+    //     Positivas: '1801', - "Positive" (infected)
+    //     Negativas: '7455', - "Negative"
+    //     FID: '1',
+    //     TOTALES: '9256', - "Total" (number tested)
+    //     FID2: '1'
+    //   }
+    // ]
+
+    // Cache this.
+    // TODO: How do we deal with multiple source for the same country?
+    // i.e If I wanted to make a Panama nation-level timeseries scraper, how would I do it
+    // and still keep this one which has greater granularity?
+    /* const timeseries = */ await fetch.csv(this._timeSeriesUrl);
+    // Array of:
+    // {
+    //   'Fecha': '2020-03-10T00:00:00.000Z', - date
+    //   TOTAL: '1', - total
+    //   Nuevos: '1', - new [cases?]
+    //   HOSPITALIZADO: '0', - hospitalized
+    //   AISLAMIENTO_DOMICILIARIO: '4', - home quarantine
+    //   FALLECIDO_: '0', - deceased
+    //   UCI: '0', - intensive care
+    //   RECUPERADO: '0', - recovered
+    //   Check: '',
+    //   FID: '1'
+    // }
+
+    // This is the one we actually get the data from.
+    const caseList = await fetch.csv(this._caseListUrl);
+    // Array of:
+    // {
+    //   'numero_CASO': '201', - case number
+    //   EDAD: '63', - age
+    //   RANGO: '61-80', - [age] range
+    //   SEXO: 'MASCULINO', - gender
+    //   FECHA_de_CONFIRMACION: '2020-03-21T00:00:00.000Z', - date of confirmation (diagnosis)
+    //   PROVINCIA: 'PANAMÁ', - province
+    //   DSITRITO: 'SAN MIGUELITO', - district (misspelled; akin to a county)
+    //   CORREGIMIENTO: 'JOSE DOMINGO ESPINAR', - this is akin to "borrough" or "neighborhood"
+    //   ESTADO_DEL_PACIENTE: 'FALLECIDO', - satus of patient
+    //          Can be FALLECIDO (deceased), ACTIVO (active), RECUPERADO (recovered)
+    //   FECHA_ACTUALIZACION: '', - update date; not sure if it's ever populated.
+    //   'UBICACIÓN_DEL_PACIENTE': 'FALLECIDO', - location of patient
+    //          Can be FALLECIDO (deceased), AISLAMIENTO DOMICILIARIO (Home quarantine),
+    //          HOSPITALIZADO (hospitalized), RECUPERADO (recovered), UCI (intensive care)
+    //   column11: '',
+    //   Edad_inicial: '',
+    //   Rango_edades: '',
+    //   FID: '1',
+    //   FID2: '1'
+    // }
+
+    const data = [];
+    log(`Panama has ${caseList.length} cumulative cases.`);
+    this.url = this._caseListUrl; // required for source rating.
+    for (const c of caseList) {
```

Comment:
possibly no need for all these abbreviations, `case` and `object` may be more immediately readable.

Reply:
Will change.